entries/s/sucuri.net.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Sucuri": {
+    "domain": "sucuri.net",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://docs.sucuri.net/billing/two-factor-authentication/",
+    "categories": [
+      "domains",
```

Comment:
The domains category is meant for domain registrars. It doesn't look like Sucuri sells domains.
```suggestion
```

Reply:
Got it. It does provide domain name servers, but yes, it's not a registrar.